src/dsp_tools/commands/excel2json/utils.py

```diff
@@ -130,6 +132,31 @@ def check_column_for_duplicate(df: pd.DataFrame, to_check_column: str) -> None |
         return None
 
 
+def check_required_values_get_position_in_excel(
```

Comment:
If the `MissingValuesProblem` contains a location attribute, then `check_required_values` should be enough, and finding the position is an implementation detail that should not be in the function name.

Reply:
I understand the comment, but this function name already exists. I will open a ticket to fix this later, so it doesn't pollute the diff here.